website/app/templates/head.hbs

```diff
@@ -0,0 +1,90 @@
+{{!
+
+Per https://github.com/shipshapecode/ember-meta#defining-your-own-headhbs this is a place where we can customize what information is injected by the ember-meta addon
+
+We've made some small customizations to simplify this compared to the default template
+
+}}
+
+{{! Begin General }}
+
+{{! TODO: Determine how to avoid this conflict between ember-meta and ember-page-title where they both set a <title> which causes the latter addon to throw an exception }}
+{{!-- {{#if this.model.title}}
+  <title>{{this.model.title}}</title>
+{{/if}} --}}
+
+{{#if this.model.description}}
+  <meta name="description" content={{this.model.description}} />
+{{/if}}
+
+<meta name="referrer" content="unsafe-url" />
+
+{{#if this.model.canonical}}
```

Comment:
where is this `canonical` value coming from? I can't see it in the code (apart from here) so my understanding is that it's never a real case